tests/cfServicesUtil.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
 import * as _ from "lodash";
 import * as sinon from "sinon";
```

Comment:
I think it is better to import it as:
import { createSandbox, SinonSandbox, SinonMock } from "sinon";

Reply:
done